src/pages/UserPage.js

```diff
@@ -218,6 +219,88 @@ class UserPage extends Component {
     );
   }
 
+  getRadarData() {
+    const userPublications = this.state.userPublications
+      ? [...this.state.userPublications]
+      : [];
+    let problemsCounter = this.state.problems ? this.state.problems.length : 0;
+
+    let hypothesesCounter = 0;

```

Comment:
The set of stages and their names should be extracted here (for production) by calling /api/problems/{id}/stages - where, for now, id would always be e.g. 0 since all current problems use the same set of stages (even though the DB and queries support different stage sets which has also been a requested feature to accommodate the diverse requirements of different branches of science)

Reply:
I agree. Good catch :+1: